lib/core/console/displayed_length.dart

```diff
@@ -0,0 +1,9 @@
+/// Removes all ANSI sequences from [str] and returns it's resulting length
+int getDisplayedLength(String str) {
+  return str.replaceAll(RegExp('\x1B[[0-9]+[a-zA-Z]'), '').length;
+}
+
+extension DisplayedLength on String {
+  /// The length of this [String] without ANSI escape sequences
+  int get displayedLength => getDisplayedLength(this);
```

Comment:
It's a proxy because I need it as a function at some point (Label.dart) to pass it to `map`.
On any other occasion an extension method is much cleaner imo. It's used in both ways.
That's for the why, if you think we should only have on way to access the function, let me know

Reply:
Makes perfect sense then :)